src/main/resources/templates/chores/index.html

```diff
@@ -23,6 +24,9 @@ <h1 th:if="${childUser}" th:text="${'THIS IS A CHILD USER'}"></h1>
         <td th:text="${chore.childAssigned.firstName}"></td>
         <td th:text="${chore.dueDay.displayName}"></td>
         <td th:text="${chore.rewardPoints}"></td>
+        <td th:text="${chore.dayOfTheWeek.displayName}"></td>
+        <td th:text="${chore.detailedDescription}"></td>
```

Comment:
```suggestion
```
Let's remove the detailed description from the list view (the idea being that it will be longer and so it probably won't fit on the table)